_includes/manuals/1.18/4.4.4_prov_ptables.md

```diff
@@ -132,6 +132,7 @@ part /     --fstype ext3 --size=1024 --grow
 part swap  --recommended
 ```
 * A script to dynamically calculate the desired sizes. E.G.
+
```

Comment:
This is the only meaningful change. The rest is trailing whitespaces which my editor removes automatically save